### PR TITLE
Fixed the difficulty in scrolling for android app

### LINKED ISSUE #737: Difficulty in scrolling of Content in Safe Paths News of Latest News screen- Android app v1.0.1
**Version**
Type of phone: Realme
OS Version: Android 9
Model: U1

**Steps to reproduce**
Open the App
Open Hamburger menu
Tap "Latest News" to navigate to SafePath News screen
Tap on Hamburger Menu against COVID SafePaths
Tap on FAQ and scroll through the contents

**Expected Behavior**
Content scrolling should be smooth and user friendly

Actual Behavior
Difficulty in scrolling of the content

### PR DESCRIPTION
**Pull request by Namit-Agrawal**
#### Description:
Closes #737 
Made scrolling more user-friendly for android users on latest news screen.

### CODE REVIEW

File: app/views/News.js [hunk 1]
```diff
@@ -129,14 +102,33 @@ class NewsScreen extends Component {
...
               }}>
-              <Carousel
-                data={this.state.newsUrls}
-                renderItem={this.renderItem}
-                sliderWidth={width}
-                itemWidth={width * 0.85}
-                layout={'default'}
-                scrollEnabled
-              />
+              <ScrollView style={{ width: width * 0.85, alignSelf: 'center' }}>
```
Comment (tstirrat) [inline app/views/News.js]: Changing out Carousel for a ScrollView seems like a big UX change. Can you provide a video of how this works now?

File: app/views/News.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ import {
   ActivityIndicator,
   BackHandler,
   Dimensions,
+  ScrollView,
   StyleSheet,
   View,
 } from 'react-native';
 import LinearGradient from 'react-native-linear-gradient';
-import Carousel from 'react-native-snap-carousel';
```
Comment (tstirrat) [inline app/views/News.js]: If this is the only place where Carousel is used, remove it from package.json too

File: app/views/News.js (hunk 1 above)
Comment (Namit-Agrawal) [reply app/views/News.js]: Sure! [Scrolling](https://youtu.be/US53QQS8kw0)

Comment (Namit-Agrawal) [reply app/views/News.js]: It is much more user-friendly now in terms of scrolling.

File: app/views/News.js (hunk 2 above)
Comment (Namit-Agrawal) [reply app/views/News.js]: Sure thing!

File: app/views/News.js (hunk 1 above)
Comment (Namit-Agrawal) [reply app/views/News.js]: Made the UI a bit more proportional for small and large screens (latest commit)

Nexus 4 screen:

Nexus4

Pixel 3 XL screen:

Pixel3XL

File: app/views/News.js [hunk 3]
```diff
@@ -127,16 +100,38 @@ class NewsScreen extends Component {
...
-                data={this.state.newsUrls}
-                renderItem={this.renderItem}
-                sliderWidth={width}
-                itemWidth={width * 0.85}
-                layout={'default'}
-                scrollEnabled
-              />
+              <ScrollView style={{ width: width * 0.85 }}>
+                {this.state.newsUrls.map((item, index) => (
+                  <View key={index} style={styles.singleNews}>
```
Comment (Patrick-Erichsen) [inline app/views/News.js]: It feels like we can pull this out into a standalone component to clean things up a bit, e.g. 

```js
{this.state.newsUrls.map((item, index) => {
  return <NewsItem item={item} index={index} />
}
```

File: app/views/News.js [hunk 4]
```diff
@@ -127,16 +100,38 @@ class NewsScreen extends Component {
...
+                    <View key={index} style={styles.singleNewsHead}>
+                      <Typography style={styles.singleNewsHeadText}>
+                        {item.name}
+                      </Typography>
+                    </View>
+                    <WebView
+                      source={{
+                        uri: item.news_url,
+                      }}
+                      style={{ height: height * 0.75 }}
```
Comment (Patrick-Erichsen) [inline app/views/News.js]: Could you pull this and the `containerStyle` out of the JSX and into the stylesheet?

File: app/views/News.js [hunk 5]
```diff
@@ -127,16 +100,38 @@ class NewsScreen extends Component {
...
+                      containerStyle={{
+                        borderBottomLeftRadius: 12,
+                        borderBottomRightRadius: 12,
+                      }}
+                      cacheEnabled
+                      onLoad={() =>
+                        this.setState({
+                          visible: false,
```
Comment (Patrick-Erichsen) [inline app/views/News.js]: small nitpick - could you change the state value to `isVisible` to better indicate its a boolean

**Comment by Namit-Agrawal** [comment]
> Hey @Namit-Agrawal , looks good for the most part! Great work.
> 
> I had a few small comments on some code cleanup. The other issue I'm seeing is that because of a somewhat long link in the FAQ, the width of the screen is now too large.
> 
> Screen Shot 2020-05-15 at 6 16 15 PM
> 
> Could you try and find a way to set a max width for the scrollview to force that URL to wrap?

Hi Patrick! It appears setting a maxwidth on the scrollview does not seem to work as we are trying 

> Hey @Namit-Agrawal , looks good for the most part! Great work.
> 
> I had a few small comments on some code cleanup. The other issue I'm seeing is that because of a somewhat long link in the FAQ, the width of the screen is now too large.
> 
> Screen Shot 2020-05-15 at 6 16 15 PM
> 
> Could you try and find a way to set a max width for the scrollview to force that URL to wrap?

Hi @Patrick-Erichsen , it appears setting a max width for the scrollview isn't working. I believe this is due to the fact that we are trying to render web content inside the scrollview. I think we would need to wrap

**Comment by Namit-Agrawal** [comment]
@tstirrat I have fixed the issue of more than 1 news item

**Pull request closed without merging**